### PR TITLE
show ↑↓ information

### PR DESCRIPTION
**Pull request by stevemao**

### CODE REVIEW

**Comment by stevemao** [comment]
Hi Paul, I have added to show show ↑↓ information about each branch's upstream, and with current branch.

screen shot 2016-08-06 at 2 29 19 pm

**Comment by stevemao** [comment]
screen shot 2016-08-06 at 2 36 25 pm

File: git-recent [hunk 1]
```diff
@@ -1,30 +1,75 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-##
-## git-recent
-##
-## list all local branches, sorted by last commit, formatted reall purdy
-##
```
Comment (cht8687) [inline git-recent:7]: I think you might need keep some comments, right? @stevemao

Comment (stevemao) [inline git-recent:7]: comments are in the bottom.

**Comment by paulirish** [comment]
It looks like the for-each-ref format has some extra sugar for this sort of thing.. I had no idea until i saw #5 

see  http://stackoverflow.com/a/20499690/89484 and https://git.kaarsemaker.net/git/commit/b28061ce0d2e31d390fee5a5724b8a061d0492a7/

`upstream:trackshort` seems pretty great. I'd be into using that and perhaps replacing the `<` `>` output with the nice arrows

but follow along with #5 as well.

**Comment by paulirish** [comment]
Closing this now as I've played with the technique in #5 more and feeling confident.

 If you're also interested in exposing remote tracking branch name, we can probably do that with a similar format fieldname.

**Comment by stevemao** [comment]
@paulirish I'll do that once #5 is merged. Cheers :)

**Comment by stevemao** [comment]
Although I don't think that approach would be able to compare other local branches with the current branch. I'm fine with that since it doesn't seem to be very useful :)

**Pull request closed without merging**